### ISSUE TITLE
Has anyone downloaded the ytb-dataset

### ISSUE THREAD

**Issue by hjjlovecyy** [songdejia/Siamese-RPN-pytorch#8]
I have VPN. And I tried to download it, but it always blocks in parse_annotations.

**Comment by zyh0904** [comment]
> I have VPN. And I tried to download it, but it always blocks in parse_annotations.

hello, have you download all data successfully? how long will it take altogether?

**Comment by hjjlovecyy** [comment]
@zyh0904 No. The dataset is too huge, so I cut down some of it, and try to download.
I add some print code so that it can let me to see what has it downloaded, and it work. But in the end, it block and never move forward.

**Comment by ucasqcz** [comment]
have you download the youtube--bb dataset ? @hjjlovecyy @zyh0904 ,can you share the baiduyun download link or download script?

**Comment by WangJerry95** [comment]
you can download the csv file manually and extract it to the correct dictionary. But the video data is too huge...

**Comment by wangaixue** [comment]
Thank you for your implement version for SiameseRPN，can you send me youtube-bb data if you download successfully, we can not vpn,the result in vid is so bad.

**Comment by salt-fisher** [comment]
the youtube-BB dataset is really huge. i have run the download.py in terminal and download 193733 videos!!!  when 1700 videos have been downloaded, about 38G have been used in disk....

**Comment by ChenSi521** [comment]
> > I have VPN. And I tried to download it, but it always blocks in parse_annotations.
> 
> hello, have you download all data successfully? how long will it take altogether?

I have modified it , so it can continue to download after it is interrupted.
[https://github.com/ChenSi521/youtube-bb-modified](url)

**Comment by csg1111** [comment]
when I run the scripts,the data folder is empty without any video？and it did not give any errorrs
have you downloaded succeedfully?
